### PR TITLE
Add charset type support

### PR DESCRIPTION
**Pull request by baev**
Add ability to auto cast `java.nio.charset.Charset`

### CODE REVIEW

File: properties-loader/src/main/java/ru/yandex/qatools/properties/converters/CharsetConverter.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+public class CharsetConverter extends AbstractConverter {
+
+    @Override
+    protected <T> T convertToType(Class<T> type, Object value) throws Throwable {
+        if (Charset.class.equals(type)) {
+            return type.cast(Charset.forName(value.toString()));
+        }
+
+        throw conversionException(type, value);
```
Comment (lanwen) [inline properties-loader/src/main/java/ru/yandex/qatools/properties/converters/CharsetConverter.java]: Why it low cased?

**Comment by baev** [comment]
@lanwen review & merge?

**Comment by lanwen** [comment]
How about build? :)

File: properties-loader/src/test/java/ru/yandex/qatools/properties/SupportedTypesTest.java [hunk 2]
```diff
@@ -90,4 +91,9 @@ public void enumTypeShouldSupported() throws Exception {
         assertThat(properties.getEnum(), equalTo(valueOf(resource.getProperty("supported.enum").toUpperCase().trim())));
     }
 
+    @Test
```
Comment (lanwen) [inline properties-loader/src/test/java/ru/yandex/qatools/properties/SupportedTypesTest.java]: add test for wrong charset pls also

**Comment by lanwen** [comment]
I'll add travis as quick fix for build, wait a bit

**Comment by baev** [comment]
@lanwen ci-qatools already build this

**Comment by baev** [comment]
http://ci.qatools.ru/view/All/job/properties_pull-request/2/

**Comment by lanwen** [comment]
perfect. Add test, then can merge

**Comment by baev** [comment]
we need more build systems!

**Pull request merged**